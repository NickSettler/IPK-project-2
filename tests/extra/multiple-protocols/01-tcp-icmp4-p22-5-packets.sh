./ipk-sniffer -i en0 --tcp --icmp4 -p 22 -n 5