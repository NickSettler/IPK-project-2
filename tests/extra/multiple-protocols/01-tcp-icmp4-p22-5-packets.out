timestamp: 2023-04-17T14:29:21.327+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 94:08:53:2e:50:cb
frame length: 98 bytes
src IP: 192.168.1.17
dst IP: 192.168.1.49

0x0000: 94 08 53 2e 50 cb b0 be 83 4a c4 b0 08 00 45 00  ..S.P....J....E.
0x0010: 00 54 64 34 00 00 40 01 92 e2 c0 a8 01 11 c0 a8  .Td4..@.........
0x0020: 01 31 08 00 4e 2d 20 b2 00 2d 64 3d 3b a1 00 04  .1..N- ..-d=;...
0x0030: fe 0d 08 09 0a 0b 0c 0d 0e 0f 10 11 12 13 14 15  ................
0x0040: 16 17 18 19 1a 1b 1c 1d 1e 1f 20 21 22 23 24 25  .......... !"#$%
0x0050: 26 27 28 29 2a 2b 2c 2d 2e 2f 30 31 32 33 34 35  &'()*+,-./012345
0x0060: 36 37                                            67

timestamp: 2023-04-17T14:29:21.399+02:00
src MAC: 94:08:53:2e:50:cb
dst MAC: b0:be:83:4a:c4:b0
frame length: 98 bytes
src IP: 192.168.1.49
dst IP: 192.168.1.17

0x0000: b0 be 83 4a c4 b0 94 08 53 2e 50 cb 08 00 45 00  ...J....S.P...E.
0x0010: 00 54 57 d9 00 00 80 01 5f 3d c0 a8 01 31 c0 a8  .TW....._=...1..
0x0020: 01 11 00 00 56 2d 20 b2 00 2d 64 3d 3b a1 00 04  ....V- ..-d=;...
0x0030: fe 0d 08 09 0a 0b 0c 0d 0e 0f 10 11 12 13 14 15  ................
0x0040: 16 17 18 19 1a 1b 1c 1d 1e 1f 20 21 22 23 24 25  .......... !"#$%
0x0050: 26 27 28 29 2a 2b 2c 2d 2e 2f 30 31 32 33 34 35  &'()*+,-./012345
0x0060: 36 37                                            67

timestamp: 2023-04-17T14:29:21.755+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 98 bytes
src IP: 2a00:1028:83a2:25ea:cc68:af55:570a:888b
dst IP: 2001:67c:1220:8b0::93e5:b013
src port: 51453
dst port: 22

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 86 dd 60 06  .w.P`^...J....`.
0x0010: 04 00 00 2c 06 40 2a 00 10 28 83 a2 25 ea cc 68  ...,.@*..(..%..h
0x0020: af 55 57 0a 88 8b 20 01 06 7c 12 20 08 b0 00 00  .UW... ..|. ....
0x0030: 00 00 93 e5 b0 13 c8 fd 00 16 2a d6 98 4f 00 00  ..........*..O..
0x0040: 00 00 b0 02 ff ff c5 6b 00 00 02 04 05 84 01 03  .......k........
0x0050: 03 06 01 01 08 0a 40 8a e0 ad 00 00 00 00 04 02  ......@.........
0x0060: 00 00                                            ..

timestamp: 2023-04-17T14:29:21.785+02:00
src MAC: 98:77:e7:50:60:5e
dst MAC: b0:be:83:4a:c4:b0
frame length: 94 bytes
src IP: 2001:67c:1220:8b0::93e5:b013
dst IP: 2a00:1028:83a2:25ea:cc68:af55:570a:888b
src port: 22
dst port: 51453

0x0000: b0 be 83 4a c4 b0 98 77 e7 50 60 5e 86 dd 60 0b  ...J...w.P`^..`.
0x0010: 4b 9b 00 28 06 34 20 01 06 7c 12 20 08 b0 00 00  K..(.4 ..|. ....
0x0020: 00 00 93 e5 b0 13 2a 00 10 28 83 a2 25 ea cc 68  ......*..(..%..h
0x0030: af 55 57 0a 88 8b 00 16 c8 fd 39 bb 9e 7f 2a d6  .UW.......9...*.
0x0040: 98 50 a0 12 7a b8 b7 f7 00 00 02 04 05 a0 04 02  .P..z...........
0x0050: 08 0a 93 ef 37 66 40 8a e0 ad 01 03 03 09        ....7f@.......

timestamp: 2023-04-17T14:29:21.785+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 86 bytes
src IP: 2a00:1028:83a2:25ea:cc68:af55:570a:888b
dst IP: 2001:67c:1220:8b0::93e5:b013
src port: 51453
dst port: 22

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 86 dd 60 06  .w.P`^...J....`.
0x0010: 04 00 00 20 06 40 2a 00 10 28 83 a2 25 ea cc 68  ... .@*..(..%..h
0x0020: af 55 57 0a 88 8b 20 01 06 7c 12 20 08 b0 00 00  .UW... ..|. ....
0x0030: 00 00 93 e5 b0 13 c8 fd 00 16 2a d6 98 50 39 bb  ..........*..P9.
0x0040: 9e 80 80 10 08 08 59 44 00 00 01 01 08 0a 40 8a  ......YD......@.
0x0050: e0 cb 93 ef 37 66                                ....7f

